common/src/main/java/org/corfudb/common/metrics/micrometer/MeterRegistryProvider.java

```diff
@@ -63,6 +100,25 @@ public static synchronized void init(Duration loggingInterval,
             init(supplier);
         }
 
+        /**
+         * Configure the meter registry of type LoggingMeterRegistry. All the metrics registered
+         * with this meter registry will be exported via provided logging sink with
+         * the provided loggingInterval frequency.
+         * @param sink A configured logging sink.
+         * @param loggingInterval A duration between log appends for every metric.
+         */
+        public static synchronized void init(Duration loggingInterval,
```

Comment:
This `init` doesn't seem like its being used.

Reply:
removed